KeePassRPC/KeePassRPCService.cs

```diff
@@ -1530,17 +1562,17 @@ public Entry[] GetChildEntries(string uuid)
 
             if (matchedGroup == null)
                 throw new Exception("Could not find requested group. Have you deleted your Kee start/home group? Set a new one and try again.");
-
-            return (Entry[])GetChildEntries(host.Database, matchedGroup, true);
+            
+            return matchedGroup;
         }
-
+        
         /// <summary>
         /// Returns a list of every entry contained within a group (not recursive)
         /// </summary>
         /// <param name="uuid">the unique ID of the group we're interested in.</param>
         /// <param name="current__"></param>
         /// <returns>the list of every entry directly inside the group.</returns>
-        private LightEntry[] GetChildEntries(PwDatabase pwd, PwGroup group, bool fullDetails)
+        private LightEntry[] GetChildEntries(PwDatabase pwd, PwGroup group, bool fullDetails, bool everyEntry)
```

Comment:
Would be clearer to flip the bool and use `urlRequired` instead - `everyEntry` would just leave people wondering which entries would be omitted if they pass in false. It's not ideal that we have to stick with the concept of "Entries" and "AllEntries" on the public API but we can at least make it clear in the private code.

Reply:
Yes. I looked into Jayrock and optional parameters - and it appears to be a mess to implement. Far simpler to expand the API, sadly.